src/utils/helpers/priorities.js

```diff
@@ -2,7 +2,16 @@ export const IMMEDIATE_PRIORITY_CODE = 1
 export const EMERGENCY_PRIORITY_CODE = 2
 export const URGENT_PRIORITY_CODE = 3
 export const NORMAL_PRIORITY_CODE = 4
+export const PLANNED_PRIORITY_CODE = 9
+export const PRIORITY_CODES_WITHOUT_DRS = [PLANNED_PRIORITY_CODE]
 
-export const LOW_PRIORITY_CODES = [URGENT_PRIORITY_CODE, NORMAL_PRIORITY_CODE]
+export const LOW_PRIORITY_CODES = [
+  URGENT_PRIORITY_CODE,
+  NORMAL_PRIORITY_CODE,
+  PLANNED_PRIORITY_CODE,
```

Comment:
Once https://github.com/LBHackney-IT/repairs-hub-frontend/pull/530 is merged, we should be able to remove this. I want us to avoid having to maintain this list every time a priority is added to the backend, and I think that PR should help us - we just use the inverse logic to detect a low priority order.